test/letters/utils/helpers.unit.spec.jsx

```diff
@@ -2,42 +2,61 @@ import { expect } from 'chai';
 import _ from 'lodash/fp';
 import SkinDeep from 'skin-deep';
 
+import { ADDRESS_TYPES, MILITARY_STATES } from '../../../src/js/letters/utils/constants';
+
 import {
-  getBenefitOptionText
-} from '../../../src/js/letters/utils/helpers.jsx';
+  getBenefitOptionText,
+  inferAddressType,
+  resetDisallowedAddressFields
+} from '../../../src/js/letters/utils/helpers';
+
+const address = {
+  countryName: 'USA',
+  addressOne: '123 Main St N',
+  stateCode: 'MA',
+  zipCode: '12345',
+  type: ADDRESS_TYPES.domestic,
+  city: 'Bygtowne'
+};
 
 describe('Letters helpers: ', () => {
   describe('getBenefitOptionText', () => {
+    // The following tests check for options that should be available / not available given certain conditions
     it('should be defined for both veterans and dependents', () => {
       _.forEach(option => {
         expect(getBenefitOptionText(option, true, true)).not.to.be.undefined;
```

Comment:
`to.exist` should ensure that the function doesn't return something that is `===` to `undefined` or `null` - should we use that as a non-negating check or do you think the current way makes more sense?

Reply:
🤔  Yeah......I don't really know what the "right" way to test these is. Maybe truthy / falsey? I'm just gonna merge it for now and we can loop back around later if it makes sense.